app/hellodjango/locations/helpers/django_admin.py

```
@@ -0,0 +1,94 @@
+            primary_number=primary_number,
+            street_name=street_name,
+            city_name=city_name,
+            state_abbreviation=state_abbreviation,
+            zip5=zip5,
+            longitude=longitude,
+            latitude=latitude,
+        )
+        us_address.save()
+        logger.info(f"Successfully created an address for {simple_address}")
+        return us_address
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)